### PR TITLE
Remove this dumb startup delay

### PR DESCRIPTION
**Pull request by mikroskeem**

### CODE REVIEW

**Review by electronicboy — APPROVED** [review]

File: BungeeCord-Patches/0056-Remove-this-dumb-startup-delay.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+diff --git a/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java b/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java
+index dc2c584d..97179044 100644
+--- a/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java
++++ b/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java
+@@ -48,11 +48,16 @@ public class BungeeCordLauncher
+             deadline.add( Calendar.WEEK_OF_YEAR, -8 );
+             if ( buildDate.before( deadline.getTime() ) )
+             {
++                /* Waterfall start - remove this dumb delay
```
Comment (aikar) [inline BungeeCord-Patches/0056-Remove-this-dumb-startup-delay.patch]: Comment is not consistent with https://github.com/PaperMC/Paper/blob/master/Spigot-Server-Patches/0163-Fix-this-stupid-bullshit.patch lol...

Comment (mikroskeem) [reply BungeeCord-Patches/0056-Remove-this-dumb-startup-delay.patch]: Will fix, sec

**Comment by Janmm14** [comment]
no, its no bullshit

**Comment by aikar** [comment]
Well we disagree lol. shouldn't punish users just because updates haven't came out in a while. We have this same change on Paper hence the patch I linked in jest to use DemonWav's original "frustration"

**Comment by Janmm14** [comment]
theres a flag to bypass delay i dont see need for this, but do what u want

**Comment by jamierocks** [comment]
If you're going to make this patch, best to merge it with https://github.com/PaperMC/Waterfall/blob/2cab85cbef7cefc63bb19ddbeb6d1bea3514a46c/BungeeCord-Patches/0034-Improve-outdated-build-message.patch.

**Comment by DoNotSpamPls** [comment]
or drop 0034 entirely

**Comment by jamierocks** [comment]
> or drop 0034 entirely

either works, but dropping the older patch would cause all patches since to have their IDs changed.

**Pull request closed without merging**